app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      def index
+        render_paginated Rent
+      end
+
+      def create
+        rent = Rent.create!(rent_params)
+        render json: rent, status: :created if rent.valid?
```

Comment:
You are running validates twice and it's not necessary.

Reply:
You can make:
```ruby
render json: rent, status: :created if Rent.create!(rent_params)
```
Or another options, there are a lot!

Reply:
@mcolombo87 in this case rent is `nil`, but its a good option